modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/CloudServiceProvider.java

```diff
@@ -17,7 +18,7 @@
  * Defines for the Cloud Service Provider.
  */
 @Data
-public class CloudServiceProvider {
+public class CloudServiceProvider implements Serializable {
 
```

Comment:
Add serialVersionUID to avoid potential incompatibilities during deserialization.